skin/adminhtml/default/openmage/menu.css

```diff
@@ -47,29 +47,29 @@
   text-align: left;
   float: left;
 }
-#nav li.over {
+#nav li:hover {
   z-index: 999;
 }
-#nav li.over a {
+#nav li:hover a {
```

Comment:
skin/adminhtml/default/openmage/scss/menu.css should also be updated :-)
(or removed?)

Reply:
Yes, true, I forget...
I'm not sure how scss works, and how to rebuild css files from scss.

Reply:
How can I rebuild the CSS from SCSS?

Reply:
it was generated with a tool named "compass" (if i'm not mistaken) which is meeega old, so in my opinion, since it's v21, we should remove the SCSS files and call it a day

Reply:
The existence of an SCSS file is good in my opinion, it is much easier to replace a value and have it be used in many places than to look for the value and replace it. Often the results are not the expected ones. Generating a css from scss is now very easy in IDEs.

Reply:
> How can I rebuild the CSS from SCSS?

With phpstorm just follow this: https://www.jetbrains.com/help/phpstorm/transpiling-sass-less-and-scss-to-css.html#install_sass_scss

> it was generated with a tool named "compass"

No, Compass was just an scss compiler - and its outdated now. Keep thar SCSS files and just use another tool to gernate CSS.

Reply:
that's what I said 🤦🏻‍♂️

Reply:
Um, you wanted to remove that SCSS file?

Reply:
@ADDISON74 with "css custom properties" we could do the same without the compiling process, modern css are almost as advanced as the preprocessors but without the need for a preprocessor. ;-)

@sreichel yes sven, you're the best and always right. please continue this way.

Reply:
ok! css regenerated, I think, not sure why `#xyz` was replaced by `hsl()`.

Reply:
"css custom properties" represents only a small part taken by CSS from SASS, which in my opinion will remain extremely powerful for a long time to come. We can take a look to CSS frameworks like Bootstrap which use SASS. I have not analyzed the scss file that is in OpenMage, if only root properties are needed then it is a solution, but SASS goes further with functions, mixins and many others.